platform/qt/src/qmapbox.cpp

```diff
@@ -24,16 +24,215 @@ static_assert(mbgl::underlying_type(QMapbox::Feature::PolygonType) == mbgl::unde
 
 namespace QMapbox {
 
+/*!
+    \namespace QMapbox
+    \inmodule Mapbox Qt SDK
+
+    Contains miscellaneous Mapbox bindings used throughout QMapboxGL.
+*/
+
+/*!
+    \typedef QMapbox::Coordinate
+
+    Synonim for QPair<double, double>.
+    Reflects mbgl::Point<double>.
+*/
+
+/*!
+    \typedef QMapbox::CoordinateZoom
+
+    Synonim for QPair<Coordinate, double>.
+    Used as return value in QMapboxGL::coordinateZoomForBounds.
+*/
+
+/*!
+    \typedef QMapbox::Coordinates
+
+    Synonim for QList<QMapbox::Coordinate>.
+    Reflects mbgl::{LineString,LinearRing,MultiPoint}<double>.
+*/
+
+/*!
+    \typedef QMapbox::CoordinatesCollection
+
+    Synonim for QList<QMapbox::Coordinates>.
+    Reflects mbgl::{MultiLineString,Polygon}<double>.
+*/
+
+/*!
+    \typedef QMapbox::CoordinatesCollections
+
+    Synonim for QList<QMApbox::CoordinatesCollection>.
+    Reflects mbgl::MultiPolygon<double>.
+*/
+
+/*!
+    \class QMapbox::Feature
+    \brief Reflects mbgl::Feature.
```

Comment:
Not sure about this. Imagine how would you interpret this if you have no idea about mbgl core.